demosplan/DemosPlanCoreBundle/Controller/Document/DemosPlanDocumentController.php

```diff
@@ -125,6 +122,7 @@ public function __construct(
      *     name="DemosPlan_plandocument_administration_element",
      *     path="/verfahren/{procedure}/verwalten/element/{elementId}",
      * )
+     *
      * @DplanPermissions("area_admin_paragraphed_document")
```

Comment:
@graupnerdemos Why is the CS-fixer now adding the empty line I wished for but couldn't get in the other PR?!